config/v1/types_cluster_version.go

```diff
@@ -379,10 +379,14 @@ const (
 	// allows to distribute Docker images
 	ClusterVersionCapabilityImageRegistry ClusterVersionCapability = "ImageRegistry"
 
-	// ClusterVersionCapabilityOperatorLifecycleManager manages the Operator Lifecycle Manager
+	// ClusterVersionCapabilityOperatorLifecycleManager manages the Operator Lifecycle Manager (legacy)
 	// which itself manages the lifecycle of operators
 	ClusterVersionCapabilityOperatorLifecycleManager ClusterVersionCapability = "OperatorLifecycleManager"
 
+	// ClusterVersionCapabilityOLMv1 manages the Operator Lifecycle Manager (v1)
+	// which itself manages the lifecycle of operators
+	ClusterVersionCapabilityOLMv1 ClusterVersionCapability = "OLMv1"
```

Comment:
No abbreviations.  It'll look silly, but it must be spelled out.

Reply:
I totally know where that's coming from. But we've had a bunch of upstream consternation on the team with the project name, now that we're trying to expand beyond operators, so we were trying to do a "Kentucky Fried Chicken" -> "KFC" sort of thing to diminish the "Operator" terminology without completely leaving behind the OLM branding.

If we go with `OperatorLifecycleManagerV1` now, but in the future determine that the "Operator" branding is hurting our ability to position the project for growing beyond operators, what would it look like to rename?